libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -455,7 +457,18 @@ protected class AuthWebViewClient extends WebViewClient {
 
         @Override
 		public void onPageFinished(WebView view, String url) {
-        	EventsObservable.get().notifyEvent(EventType.AuthWebViewPageFinished, url);
+            // Remove the Biometric Login button from the Connected App allow denny screen.
+            if (url.contains("frontdoor.jsp")) {
+                final RelativeLayout parentView = (RelativeLayout) view.getParent();
+                if (parentView != null) {
+                    final Button button = parentView.findViewById(R.id.sf__bio_login_button);
+                    if (button != null) {
+                        button.setVisibility(View.INVISIBLE);
+                    }
+                }
+            }
```

Comment:
There is likely a cleaner way to do this, will try to improve it.  Suggestions welcome.

Reply:
I'll probably have to do something similar with the "Login with IDP" button.

Reply:
So if you reload the web view from the menu, will the button reappear?

Reply:
`onPageFinished` gets called again if you reload from the menu.